web/src/lib/stores/assets.store.ts

```diff
@@ -38,6 +38,7 @@ export class AssetBucket {
   assets!: AssetResponseDto[];
   cancelToken!: AbortController | null;
   position!: BucketPosition;
+  preventCancel!: boolean;
```

Comment:
Why do we need this mechanism?

Reply:
In order to scroll to an asset, the bucket must be loaded. Offscreen buckets are automatically canceled. Here, preventCancel is used to force a bucket to load, even if offscreen, so that it can be scrolled to after the bucket is loaded.